pipeline/test/services/workload-cluster/testHNC.sh

```diff
@@ -43,7 +43,8 @@ function check_wc_hnc_creation_removal() {
     debug_msg=""
 
     mapfile -t user_namespaces < <(yq4 -e '.user.namespaces[]' "${config['config_file_wc']}")
-    mapfile -t user_admin_users < <(yq4 -e '.user.adminUsers[]' "${config['config_file_wc']}")
+    mapfile -t user_admin_users < <(yq4 -e '.user.adminUsers[] // ""' "${config['config_file_wc']}")
```

Comment:
Why was this changed? I get this output as a result
![image](https://github.com/user-attachments/assets/f85aba75-aeaa-42bd-8067-0c5731b19b87)

Reply:
Because if the list was empty it would fail for me. But this is not the behaviour I'm seeing, I guess it might be some yq or bash version difference :thinking: